lib/pages/authen/register/widgets/full_name_form.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:structure_flutter/bloc/bloc.dart';
+import 'package:structure_flutter/core/resource/icon_style.dart';
+import 'package:structure_flutter/core/resource/text_style.dart';
+import 'package:structure_flutter/di/injection.dart';
+
+class FullNameForm extends StatefulWidget {
+  final RegisterState _state;
+  final TextEditingController _fullNameController;
+
+  FullNameForm(this._state, this._fullNameController);
+
+  @override
+  _PasswordFormState createState() => _PasswordFormState();
+}
+
+class _PasswordFormState extends State<FullNameForm> {
```

Comment:
```suggestion
class _FullNameState extends State<FullNameForm> {
```
State's name should be match with Widget's name (kill) (kill)